### PR TITLE
Organize parameter names

### PR DESCRIPTION
**Pull request by 0xMuluh**
This commit organizes parameters used by user.

### CODE REVIEW

File: R/agglomerate.R [hunk 1]
```diff
@@ -242,8 +252,9 @@ setGeneric("agglomerateByVariable",
 #'
 #' @export
 setMethod("agglomerateByRank", signature = c(x = "SummarizedExperiment"),
-    function(x, rank = taxonomyRanks(x)[1], onRankOnly = TRUE, na.rm = FALSE,
-        empty.fields = c(NA, "", " ", "\t", "-", "_"), ...){
+    function(x, rank = taxonomyRanks(x)[1], onRankOnly= TRUE, na.rm = FALSE, 
+             empty.fields = c(NA, "", " ", "\t", "-", "_"), disable.taxonomy = onRankOnly,
```
Comment (TuomasBorman) [inline R/agglomerate.R]: The name parameter could be there first, next to old. It might be hard to see. The order is same in documentation

-->

disable.taxonomy = onRankOnly, onRankOnly = TRUE

Also check that the indentation is correct in lines that you add (multiplication of 4) https://contributions.bioconductor.org/r-code.html?q=indent#indentation

File: R/agglomerate.R [hunk 2]
```diff
@@ -23,9 +23,11 @@
 #' @param rank a single character defining a taxonomic rank. Must be a value of
 #'   \code{taxonomyRanks()} function.
 #'
-#' @param onRankOnly \code{TRUE} or \code{FALSE}: Should information only from
+#' @param disable.taxonomy \code{TRUE} or \code{FALSE}: Should information only from
 #'   the specified rank be used or from ranks equal and above? See details.
-#'   (default: \code{onRankOnly = FALSE})
```
Comment (TuomasBorman) [inline R/agglomerate.R]: Remember to add

@param onRankOnly Deprecated. Please use \code{disable.taxonomy} instead.

File: R/importQIIME2.R [hunk 3]
```diff
@@ -68,57 +80,63 @@
...
                             taxonomyTableFile = NULL,
+                            col.file = sampleMetaFile,
                             sampleMetaFile = NULL,
+                            as.refseq = featureNamesAsRefSeq,
                             featureNamesAsRefSeq = TRUE,
+                            refseq.file = refSeqFile,
                             refSeqFile = NULL,
+                            phy.tree.file = phyTreeFile,
                             phyTreeFile = NULL,
+                            assay.file = featureTableFile,
```
Comment (TuomasBorman) [inline R/importQIIME2.R]: Also here the order should be modified. assay.file must be the first parameter

File: R/agglomerate.R (hunk 2 above)
Comment (0xMuluh) [reply R/agglomerate.R]: the @param is on line 30

File: R/agglomerate.R (hunk 1 above)
Comment (0xMuluh) [reply R/agglomerate.R]: Ahh was browsing some failing tests then forgot to reorder it. its fixed now.

File: R/importQIIME2.R (hunk 3 above)
Comment (0xMuluh) [reply R/importQIIME2.R]: ACK

File: R/calculateUnifrac.R [hunk 4]
```diff
@@ -212,7 +216,7 @@ setMethod("calculateUnifrac",
 #'
 #' @export
 runUnifrac <- function(x, tree, weighted = FALSE, normalized = TRUE,
-                       nodeLab = NULL, BPPARAM = SerialParam(), ...){
+                       node.lab = nodeLab, nodeLab = NULL, BPPARAM = SerialParam(), ...){
```
Comment (TuomasBorman) [inline R/calculateUnifrac.R]: Is the indentation correct? Might be

**Comment by TuomasBorman** [comment]
Should we use MARGIN as a parameter name for instance in agglomerateByVariable https://github.com/microbiome/mia/pull/556

File: R/agglomerate.R [hunk 5]
```diff
@@ -88,18 +94,22 @@
 #'   as \code{levels(f)}. (Default: \code{archetype = 1L}, which means the first
 #'   element encountered per factor level will be kept)
 #'
-#' @param mergeTree \code{TRUE} or \code{FALSE}: Should
-#'   \code{rowTree()} also be merged? (Default: \code{mergeTree = FALSE})
+#' @param agglomerate.tree \code{TRUE} or \code{FALSE}: Should
+#'   \code{rowTree()} also be merged? (Default: \code{agglomerate.tree = FALSE})
 #'
```
Comment (TuomasBorman) [inline R/agglomerate.R]: Wondering if agglomerate.* parameters are too verbose. Should they be agg.* for instance agg.tree

File: R/estimateDiversity.R [hunk 6]
```diff
@@ -586,22 +591,22 @@ setMethod("estimateFaith", signature = c(x="TreeSummarizedExperiment", tree="mis
     return(faiths)
 }
 
-.calc_log_modulo_skewness <- function(mat, quantile = 0.5, num_of_classes = 50, ...){
+.calc_log_modulo_skewness <- function(mat, quantile = 0.5, nclasses = 50, ...){
```
Comment (TuomasBorman) [inline R/estimateDiversity.R:594]: nclasses was chose to be the name because we have also parameter ntop which can also be found from scater package

File: R/importHumann.R [hunk 7]
```diff
@@ -70,7 +72,7 @@
 #' 
 NULL
 
-importHUMAnN <- function(file, colData = NULL, ...){
+importHUMAnN <- function(file, col.data = colData, colData = NULL, ...){
```
Comment (TuomasBorman) [inline R/importHumann.R:75]: colData is used in constructor functions, but this is not constructor

File: R/importHumann.R [hunk 8]
```diff
@@ -3,19 +3,21 @@
...
 #' @param ... additional arguments:
 #' \itemize{
 #'   \item \code{assay.type}: A single character value for naming 
 #'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{assay}} 
 #'   (default: \code{assay.type = "counts"})
-#'   \item \code{removeTaxaPrefixes}: \code{TRUE} or \code{FALSE}: Should
+#'   \item \code{remove.taxa.prefix}: \code{TRUE} or \code{FALSE}: Should
 #'     taxonomic prefixes be removed? (default:
-#'     \code{removeTaxaPrefixes = FALSE})
+#'     \code{remove.taxa.prefix = FALSE})
```
Comment (TuomasBorman) [inline R/importHumann.R]: The name could be also taxa.prefix.rm. Not sure if that is any better (the name is quite long)

File: R/importMetaphlan.R [hunk 9]
```diff
@@ -3,19 +3,25 @@
...
+#'   (default: \code{col.data = NULL}).
+#'   
+#' @param colData Deprecated. use \code{col.data} instead.
 #' 
-#' @param sample_meta a DataFrame-like object that includes sample names in
+#' @param col.data a DataFrame-like object that includes sample names in
 #'   rownames, or a single \code{character} value defining the file
 #'   path of the sample metadata file. The file must be in \code{tsv} format
-#'   (default: \code{sample_meta = NULL}).
+#'   (default: \code{col.data = NULL}).
```
Comment (TuomasBorman) [inline R/importMetaphlan.R]: Remove this since this is duplicated

File: R/importMetaphlan.R [hunk 10]
```diff
@@ -3,19 +3,25 @@
...
+#'   
+#' @param sample_meta Deprecated. Use \code{col.data} instead.
 #' 
-#' @param phy_tree a single \code{character} value defining the file
+#' @param phy.tree.file a single \code{character} value defining the file
```
Comment (TuomasBorman) [inline R/importMetaphlan.R:15]: file suffix was added to denote that this is a file. Not object.

File: R/importMothur.R [hunk 11]
```diff
@@ -3,25 +3,25 @@
...
-#' @param taxonomyFile a single \code{character} value defining the file path of
+#' @param row.file a single \code{character} value defining the file path of
 #'   the taxonomy table to be imported. The File has to be in \code{taxonomy
 #'   file} or \code{constaxonomy file} format  as defined in Mothur
-#'   documentation. (default: \code{taxonomyFile = NULL}).
+#'   documentation. (default: \code{row.file = NULL}).
 #'
-#' @param designFile a single \code{character} value defining the file path of
+#' @param col.file a single \code{character} value defining the file path of
 #'   the sample metadata to be imported. The File has to be in \code{desing
```
Comment (TuomasBorman) [inline R/importMothur.R]: Capital F in File

File: R/importMothur.R [hunk 12]
```diff
@@ -63,66 +63,66 @@ NULL
 #' @importFrom SummarizedExperiment SummarizedExperiment
 #' @importFrom S4Vectors make_zero_col_DFrame
 #' @export
-importMothur <- function(sharedFile,
-                            taxonomyFile = NULL,
-                            designFile = NULL) {
+importMothur <- function(assay.file,
+                            row.file = NULL,
+                            col.file = NULL) {
```
Comment (TuomasBorman) [inline R/importMothur.R]: You did not deprecate old names

Comment (TuomasBorman) [reply R/importMothur.R]: Also they are missing from documentation (the old names)

File: R/importQIIME2.R [hunk 13]
```diff
@@ -68,57 +80,63 @@
...
 #' @examples
-#' featureTableFile <- system.file("extdata", "table.qza", package = "mia")
-#' taxonomyTableFile <- system.file("extdata", "taxonomy.qza", package = "mia")
-#' sampleMetaFile <- system.file("extdata", "sample-metadata.tsv", package = "mia")
-#' phyTreeFile <- system.file("extdata", "tree.qza", package = "mia")
-#' refSeqFile <- system.file("extdata", "refseq.qza", package = "mia")
+#' assay.file <- system.file("extdata", "table.qza", package = "mia")
+#' row.file <- system.file("extdata", "taxonomy.qza", package = "mia")
+#' col.file <- system.file("extdata", "sample-metadata.tsv", package = "mia")
+#' phy.tree.file <- system.file("extdata", "tree.qza", package = "mia")
```
Comment (TuomasBorman) [inline R/importQIIME2.R]: Can you convert all phy.tree.file parameter names to tree.file. That is more simple

Comment (TuomasBorman) [reply R/importQIIME2.R]: Otherwise, I think this naming assay.file, col.file... is good, but other opinions are very welcome

File: R/makeTreeSummarizedExperimentFromBiom.R [hunk 14]
```diff
@@ -74,32 +78,32 @@ importBIOM <- function(file, ...) {
...
 #'
-#' @param obj object of type \code{\link[biomformat:read_biom]{biom}}
+#' @param x object of type \code{\link[biomformat:read_biom]{biom}}
 #'
 #' @export
 #' @importFrom S4Vectors make_zero_col_DFrame DataFrame
 #' @importFrom dplyr %>% bind_rows
-makeTreeSEFromBiom <- function(
-        obj, removeTaxaPrefixes = FALSE, rankFromPrefix = FALSE,
+makeTreeSEFromBiom <- function(x , remove.taxa.prefix = removeTaxaPrefixes,
```
Comment (TuomasBorman) [inline R/makeTreeSummarizedExperimentFromBiom.R]: Remove space after x

File: R/merge.R [hunk 15]
```diff
@@ -196,24 +196,24 @@
     seq
 }
 
-.merge_rows_TSE <- function(x, f, archetype = 1L, mergeTree = FALSE,
-                            mergeRefSeq = FALSE, ...){
+.merge_rows_TSE <- function(x, f, archetype = 1L, agglomerate.tree = FALSE,
+                            agglomerate.refseq = FALSE, ...){
```
Comment (TuomasBorman) [inline R/merge.R:192]: Have you taken into account deprecation (it might that old names are not needed here if the function that is calling these is taken care of that) --> check

File: R/merge.R [hunk 16]
```diff
@@ -223,15 +223,15 @@
...
 
-.merge_cols_TSE <- function(x, f, archetype = 1L, mergeTree = FALSE, ...){
+.merge_cols_TSE <- function(x, f, archetype = 1L, agglomerate.tree = FALSE, ...){
     # input check
-    if(!.is_a_bool(mergeTree)){
-        stop("'mergeTree' must be TRUE or FALSE.", call. = FALSE)
+    if(!.is_a_bool(agglomerate.tree)){
+        stop("'agglomerate.tree' must be TRUE or FALSE.", call. = FALSE)
     }
     #
```
Comment (TuomasBorman) [inline R/merge.R:223]: Same here. Check if deprecation is taken care of. (unit tests should reveal that)

File: R/rarefyAssay.R [hunk 17]
```diff
@@ -27,10 +27,12 @@
 #'   (Please use \code{assay.type} instead. At some point \code{assay_name}
 #'   will be disabled.)
 #'   
-#' @param min_size A single integer value equal to the number of counts being 
+#' @param sample A single integer value equal to the number of counts being 
 #'   simulated this can equal to lowest number of total counts 
 #'   found in a sample or a user specified number. 
 #'
```
Comment (TuomasBorman) [inline R/rarefyAssay.R:33]: The name "sample" was chose because vegan uses the same naming. However, phyloseq uses min_size...

But one important point is that when rarefied beta diversities are calculated, one should use avgdist() function from vegan (and that has sample parameter)

File: R/runDPCoA.R [hunk 18]
```diff
@@ -91,7 +95,7 @@ setGeneric("calculateDPCoA", signature = c("x", "y"),
            function(x, y, ...)
                standardGeneric("calculateDPCoA"))
 
-.calculate_dpcoa <- function(x, y, ncomponents = 2, ntop = NULL,
+.calculate_dpcoa <- function(x, y, ncomponents = 2, ntop = NULL, subset.row = subset_row,
```
Comment (TuomasBorman) [inline R/runDPCoA.R]: Is this line under 80 characters long? Bioc guidelines https://contributions.bioconductor.org/r-code.html?q=80%20cha#indentation

File: R/splitByRanks.R [hunk 19]
```diff
@@ -19,10 +19,12 @@
 #'   will be dropped. This setting can be tweaked by defining
 #'   \code{empty.fields} to your needs. (default: \code{na.rm = TRUE})
 #'
-#' @param keep_reducedDims \code{TRUE} or \code{FALSE}: Should the
+#' @param keep.reduceddims \code{TRUE} or \code{FALSE}: Should the
```
Comment (TuomasBorman) [inline R/splitByRanks.R:22]: Capital D was removed so that the naming is harmonized. We have only lowercase characters in parameters (except *.FUN which is established way to denote function)

File: R/splitOn.R [hunk 20]
```diff
@@ -275,19 +283,19 @@ setMethod("splitOn", signature = c(x = "SingleCellExperiment"),
 #' @rdname splitOn
 #' @export
 setMethod("splitOn", signature = c(x = "TreeSummarizedExperiment"),
-    function(x, f = NULL, update_rowTree = FALSE,
+    function(x, f = NULL, update.rowtree = update_rowTree, update_rowTree = FALSE,
```
Comment (TuomasBorman) [inline R/splitOn.R]: Same here. The name does not have uppercase characters anymore

File: R/transformCounts.R [hunk 21]
```diff
@@ -33,9 +33,10 @@
 #'
 #' @param ... additional arguments passed on to \code{vegan:decostand}:
 #' \itemize{
-#'   \item \code{ref_vals}: A single value which will be used to fill 
+#'   \item \code{reference}: A single value which will be used to fill
```
Comment (TuomasBorman) [inline R/transformCounts.R:36]: reference name is used in vegan

Comment (TuomasBorman) [reply R/transformCounts.R:36]: In a function that calculates ALR

File: R/utils.R [hunk 22]
```diff
@@ -80,25 +80,25 @@
...
-                                   name = .get_name_in_parent(tree_name) ){
-    if( !.is_non_empty_string(tree_name) ){
+.check_rowTree_present <- function(tree.name, x,
+                                   name = .get_name_in_parent(tree.name) ){
+    if( !.is_non_empty_string(tree.name) ){
         stop("'", name, "' must be a single non-empty character value.",
              call. = FALSE)
     }
-    if( !(tree_name %in% names(x@rowTree)) ){
+    if( !(tree.name %in% names(x@rowTree)) ){
```
Comment (TuomasBorman) [inline R/utils.R]: At the same go, can you fix this line. @ should not be used. Instead, one should use getters of the class. In this case

names(x@rowTree) --> rowTreeNames(x)

File: R/utils.R [hunk 23]
```diff
@@ -80,25 +80,25 @@
...
-                                   name = .get_name_in_parent(tree_name) ){
-    if( !.is_non_empty_string(tree_name) ){
+.check_colTree_present <- function(tree.name, x,
+                                   name = .get_name_in_parent(tree.name) ){
+    if( !.is_non_empty_string(tree.name) ){
         stop("'", name, "' must be a single non-empty character value.",
              call. = FALSE)
     }
-    if( !(tree_name %in% names(x@colTree)) ){
+    if( !(tree.name %in% names(x@colTree)) ){
```
Comment (TuomasBorman) [inline R/utils.R]: Same here --> colTreeNames(x)

File: man/Tengeler2020.Rd [hunk 24]
```diff
@@ -14,7 +14,7 @@ Family and Genus level. The colData includes:
 \item{cohort}{cohort to which the patient belongs (Cohort_1, Cohort_2 and
 Cohort_3)}
 \item{patient_status_vs_cohort}{combination of patient_status and cohort}
-\item{sample_name}{unique sample ID}
+\item{col.name}{unique sample ID}
```
Comment (TuomasBorman) [inline man/Tengeler2020.Rd]: I think this has not changed???

Comment (TuomasBorman) [reply man/Tengeler2020.Rd]: In reality. This is descrption of example  data

**Comment by TuomasBorman** [comment]
The fail of check is not related to this PR

**Comment by TuomasBorman** [comment]
Does this already take into account all parameters from the table?

File: R/agglomerate.R (hunk 5 above)
Comment (antagomir) [reply R/agglomerate.R]: It is a bit verbose, on the other hand it is very clear. I might have a slight preference for the full name but no strong opinions. If shorter name is used then it could also be "aggl.*"

File: R/importHumann.R (hunk 8 above)
Comment (antagomir) [reply R/importHumann.R]: or `rm.prefix` - the purpose might be clear from the context, and rm is widely used abbreviation. Although remove.taxa.prefix is more clear.

Comment (antagomir) [reply R/importHumann.R]: i would do rm.* rather than *.rm (think about rm.na for instance, let's stay consistent)

File: R/splitByRanks.R (hunk 19 above)
Comment (antagomir) [reply R/splitByRanks.R:22]: should it be keep.reddim? I think reddim was used somewhere else, too. Perhaps scater.

File: R/agglomerate.R (hunk 5 above)
Comment (0xMuluh) [reply R/agglomerate.R]: Should we maintain or perhaps change from agglomerate.* to agg.*?

File: R/calculateUnifrac.R (hunk 4 above)
Comment (0xMuluh) [reply R/calculateUnifrac.R]: ACK

File: R/importMothur.R (hunk 12 above)
Comment (0xMuluh) [reply R/importMothur.R]: when i deprecate old name here, function is unable to locate file.

File: R/importHumann.R (hunk 8 above)
Comment (0xMuluh) [reply R/importHumann.R]: so rm.*?

File: R/importMetaphlan.R (hunk 9 above)
Comment (0xMuluh) [reply R/importMetaphlan.R]: ACK.

File: R/importMetaphlan.R (hunk 10 above)
Comment (Daenarys8) [reply R/importMetaphlan.R:20]: ACK.

File: R/importMothur.R (hunk 11 above)
Comment (0xMuluh) [reply R/importMothur.R]: ACK.

File: R/importQIIME2.R (hunk 13 above)
Comment (0xMuluh) [reply R/importQIIME2.R]: ACK.

File: R/makeTreeSummarizedExperimentFromBiom.R (hunk 14 above)
Comment (0xMuluh) [reply R/makeTreeSummarizedExperimentFromBiom.R]: ACK.

File: R/merge.R (hunk 15 above)
Comment (0xMuluh) [reply R/merge.R:192]: yes, changed old names in internal functions because i deprecated in functions that call the.

File: R/merge.R (hunk 16 above)
Comment (0xMuluh) [reply R/merge.R:223]: ACK.

File: R/runDPCoA.R (hunk 18 above)
Comment (0xMuluh) [reply R/runDPCoA.R]: ACK.

**Comment by 0xMuluh** [comment]
> Does this already take into account all parameters from the table?

yes. Except for obj parameter in summary. For some reason it rejects and throws error about signature.

File: R/agglomerate.R (hunk 5 above)
Comment (TuomasBorman) [reply R/agglomerate.R]: We can keep them as agglomerate*

File: R/importHumann.R (hunk 8 above)
Comment (TuomasBorman) [reply R/importHumann.R]: I did not get the point of rm.*

na.rm is widely used in R, so based on that, the justification is for prefix.rm

File: man/Tengeler2020.Rd (hunk 24 above)
Comment (TuomasBorman) [reply man/Tengeler2020.Rd]: So please revert this

**Comment by TuomasBorman** [comment]
Btw use normal push instead of force push for common push operation.

This might be hazardous if someone else is also working in this PR. You are forcing the history be the same as in your local. Force push should not be the ordinary tool to use

**Comment by TuomasBorman** [comment]
> > Does this already take into account all parameters from the table?
> 
> yes. Except for obj parameter in summary. For some reason it rejects and throws error about signature.

You should change also the signature
`
signature = c(object = "SummarizedExperiment") `--> `signature = c(x = "SummarizedExperiment")`

File: R/splitByRanks.R (hunk 19 above)
Comment (TuomasBorman) [reply R/splitByRanks.R:22]: Scater and scuttle use  `dimred` very widely. This could be keep.dimred

**Comment by 0xMuluh** [comment]
> > > Does this already take into account all parameters from the table?
> > 
> > 
> > yes. Except for obj parameter in summary. For some reason it rejects and throws error about signature.
> 
> You should change also the signature `signature = c(object = "SummarizedExperiment")`--> `signature = c(x = "SummarizedExperiment")`

I did change the signature. Then it issued a warning about :
 ```
Warning message:
multiple methods tables found for ‘summary’ 
```

**Comment by TuomasBorman** [comment]
Ahh, it seems that generic summary function comes from R ecosystem. We have not defined summary as generic function, we have only defined methods for summary function. That R ecosystem summary uses "object" in its signature. So this problem comes from there.

We can keep the argument name as object in this case

File: R/agglomerate.R (hunk 5 above)
Comment (0xMuluh) [reply R/agglomerate.R]: ACK.

File: R/importHumann.R (hunk 8 above)
Comment (0xMuluh) [reply R/importHumann.R]: ACK.

File: man/Tengeler2020.Rd (hunk 24 above)
Comment (0xMuluh) [reply man/Tengeler2020.Rd]: ACK.

File: R/calculateUnifrac.R [hunk 25]
```diff
@@ -234,28 +238,28 @@ runUnifrac <- function(x, tree, weighted = FALSE, normalized = TRUE,
...
-             "'nodeLab'.", call. = FALSE)
+             "'node.lab'.", call. = FALSE)
     }
     # Merge rows, so that rows that are assigned to same tree node are agglomerated
     # together. If nodeLabs were provided, merge based on those. Otherwise merge
     # based on rownames
-    if( is.null(nodeLab) ){
-        nodeLab <- rownames(x)
+    if( is.null(node.lab) ){
+      node.lab <- rownames(x)
```
Comment (TuomasBorman) [inline R/calculateUnifrac.R]: indentation is wrong

File: R/importMothur.R [hunk 26]
```diff
@@ -63,66 +69,69 @@ NULL
 #' @importFrom SummarizedExperiment SummarizedExperiment
 #' @importFrom S4Vectors make_zero_col_DFrame
 #' @export
-importMothur <- function(sharedFile,
-                            taxonomyFile = NULL,
-                            designFile = NULL) {
+importMothur <- function(sharedFile,
```
Comment (TuomasBorman) [inline R/importMothur.R]: assay.file should be first since sharedFile will be deprecated

File: R/utils.R [hunk 27]
```diff
@@ -357,19 +357,19 @@
...
                 "The sample ids in feature table are not incompatible ",
                 "with those in sample meta",
                 call. = FALSE
             )
         }
-        if (!identical(colnames(feature_tab), rownames(sample_meta))) {
-            feature_tab <- feature_tab[, rownames(sample_meta), drop = FALSE]
+        if (!identical(colnames(feature_tab), rownames(col.data))) {
+            feature_tab <- feature_tab[, rownames(col.data), drop = FALSE]
         }
```
Comment (TuomasBorman) [inline R/utils.R]: No need to change these internal variables

File: R/splitOn.R [hunk 28]
```diff
@@ -13,26 +13,33 @@
...
-#' @param update_rowTree \code{TRUE} or \code{FALSE}: Should the rowTree be updated
+#' @param keep_reducedDims Deprecated. Use \code{keep.reduceddims} instead.
+#'   
+#' @param update.rowtree \code{TRUE} or \code{FALSE}: Should the rowTree be updated
 #'   based on splitted data? Option is enabled when \code{x} is a 
 #'   \code{TreeSummarizedExperiment} object or a list of such objects. 
-#'   (By default: \code{update_rowTree = FALSE})
+#'   (By default: \code{update.rowtree = FALSE})
+#'   
+#' @param update_rowTree Deprecated. Use \code{update.rowtree} isntead.
```
Comment (TuomasBorman) [inline R/splitOn.R]: typo: isntead

File: R/splitOn.R [hunk 29]
```diff
@@ -13,26 +13,33 @@
...
+#' @param keep.reduceddims \code{TRUE} or \code{FALSE}: Should the
 #'   \code{reducedDims(x)} be transferred to the result? Please note, that this
 #'   breaks the link between the data used to calculate the reduced dims.
-#'   (By default: \code{keep_reducedDims = FALSE})
+#'   (By default: \code{keep.reduceddims = FALSE})
 #'   
-#' @param update_rowTree \code{TRUE} or \code{FALSE}: Should the rowTree be updated
+#' @param keep_reducedDims Deprecated. Use \code{keep.reduceddims} instead.
+#'   
+#' @param update.rowtree \code{TRUE} or \code{FALSE}: Should the rowTree be updated
```
Comment (TuomasBorman) [inline R/splitOn.R]: This could be agglomerate.tree to harmonize between functions. agglomerate* functions use agglomerate.tree name. Here the parameter functions similarly

**Comment by TuomasBorman** [comment]
@antagomir Thoughts? I think the naming convention is now good with these changes.

MARGIN is the only naming-thing left https://github.com/microbiome/mia/pull/556

File: R/agglomerate.R [hunk 30]
```diff
@@ -49,17 +51,19 @@
 #'   to \code{getPrevalence} and \code{getPrevalentTaxa} and used in
 #'   \code{agglomeratebyPrevalence}
 #'   \itemize{
-#'        \item \code{remove_empty_ranks}: A single boolean value for selecting 
+#'        \item \code{empty.ranks.rm}: A single boolean value for selecting
```
Comment (TuomasBorman) [inline R/agglomerate.R]: This is not actually removing empty ranks. This is for removing empty columns from rowData (column can be anything, also those that are not ranks). The parameter name is ok, but this behavior should be changed. (only rank colum should be removed). That modification must be done on other PR.

File: R/agglomerate.R [hunk 31]
```diff
@@ -23,9 +23,11 @@
 #' @param rank a single character defining a taxonomic rank. Must be a value of
 #'   \code{taxonomyRanks()} function.
 #'
-#' @param onRankOnly \code{TRUE} or \code{FALSE}: Should information only from
+#' @param disable.taxonomy \code{TRUE} or \code{FALSE}: Should information only from
```
Comment (TuomasBorman) [inline R/agglomerate.R]: This parameter could be removed. It does not give any extra value. When onRankOnly =TRUE, then this function works like agglomeratebyVariable. (This is also something that must be done in another PR)

File: R/splitOn.R [hunk 32]
```diff
@@ -386,7 +394,7 @@ setGeneric("unsplitOn",
     # IF the object is TreeSE. add rowTree
     if( class_x == "TreeSummarizedExperiment" ){
         # Update or add old tree from the first element of list
-        if( update_rowTree ){
+        if( agglomerate.tree  ){
             ans <- addHierarchyTree(ans)
```
Comment (TuomasBorman) [inline R/splitOn.R:398]: I haven't seen this before, but this is a bug. It calculates new tree instead of pruning the existing one.

File: R/agglomerate.R (hunk 31 above)
Comment (TuomasBorman) [reply R/agglomerate.R]: https://github.com/microbiome/mia/issues/559

File: R/splitOn.R (hunk 32 above)
Comment (TuomasBorman) [reply R/splitOn.R:398]: https://github.com/microbiome/mia/issues/569

File: R/agglomerate.R (hunk 30 above)
Comment (TuomasBorman) [reply R/agglomerate.R]: https://github.com/microbiome/mia/issues/568

File: R/calculateUnifrac.R [hunk 33]
```diff
@@ -121,19 +125,19 @@ setMethod("calculateUnifrac",
         signature = c(x = "TreeSummarizedExperiment",
                     tree = "missing"),
     function(x, assay.type = assay_name, assay_name = exprs_values, exprs_values = "counts", 
-            tree_name = "phylo", transposed = FALSE, ...){
+             tree.name = tree_name, tree_name = "phylo", transposed = FALSE, ...){
```
Comment (TuomasBorman) [inline R/calculateUnifrac.R:128]: INdentation seems to be off, run BiocCheck::BiocCheck() and check every line that is modified in this PR

File: R/calculateUnifrac.R [hunk 34]
```diff
@@ -162,22 +166,27 @@ setMethod("calculateUnifrac",
             links <- colLinks(x)
         }
         # Remove those links (make them NA) that are not included in this tree
-        links[ links$whichTree != tree_name, ] <- NA
+        links[ links$whichTree != tree.name, ] <- NA
         # Take only nodeLabs
         links <- links[ , "nodeLab" ]
-        res <- calculateUnifrac(mat, tree = tree, nodeLab = links, ...)
-        return(res)
+        calculateUnifrac(mat, tree = tree, node.lab = links, ...)
```
Comment (TuomasBorman) [inline R/calculateUnifrac.R:172]: It seems that your PR has removed all the changes in devel branch. This line has been changed

File: R/calculateUnifrac.R (hunk 33 above)
Comment (TuomasBorman) [reply R/calculateUnifrac.R:128]: I think you can manually revert these lines. There are not too many

File: R/calculateUnifrac.R (hunk 34 above)
Comment (TuomasBorman) [reply R/calculateUnifrac.R:172]: I think you can manually revert your changes regarding changes that removed modification from devel

Comment (TuomasBorman) [reply R/calculateUnifrac.R:172]: there are not too many

**Review by TuomasBorman — CHANGES_REQUESTED** [review]
Thanks, please check the unifrac method

**Comment by TuomasBorman** [comment]
Closing since duplicated with https://github.com/microbiome/mia/pull/582
